m2m-transfers/V3/REPWRITERSPECS/BANNO.M2MTRANSFERS.V3.POW

```diff
@@ -2786,10 +2788,12 @@ PROCEDURE ERRORHANDLER
  ERRORDETAILOFFSET=ERRORCODE-500
 
  IF ERRORDETAILOFFSET=1 THEN
-  IF READCONFIGERROR=TRUE THEN
-   ERRORDETAIL(1)=ERRORDETAIL(1)+": "+READCONFIGERRORMSG
-  ELSE IF PARAMVALIDATIONERROR=TRUE THEN
-   ERRORDETAIL(1)=ERRORDETAIL(1)+": Invalid Parameter in CFG file"
+  DO
+   IF READCONFIGERROR=TRUE THEN
+    ERRORDETAIL(1)=ERRORDETAIL(1)+": "+READCONFIGERRORMSG
+   ELSE IF PARAMVALIDATIONERROR=TRUE THEN
+    ERRORDETAIL(1)=ERRORDETAIL(1)+": Invalid Parameter in CFG file"
+  END
 
  IF ERRORDETAILOFFSET=2 THEN
```

Comment:
Suggest you do the DO/END block here, too.

Reply:
Thanks @rrobison44 . I've added the last change, and will merge the code now.